receiver/otlpreceiver/metrics/otlp.go

```diff
@@ -57,8 +57,14 @@ const (
 func (r *Receiver) Export(ctx context.Context, req *collectormetrics.ExportMetricsServiceRequest) (*collectormetrics.ExportMetricsServiceResponse, error) {
 	receiverCtx := obsreport.ReceiverContext(ctx, r.instanceName, receiverTransport, receiverTagValue)
 
-	for _, resourceMetrics := range req.ResourceMetrics {
-		err := r.processReceivedMetrics(receiverCtx, resourceMetrics)
+	md := data.MetricDataFromOtlp(req.ResourceMetrics)
+	rms := md.ResourceMetrics()
+	for i := 0; i < rms.Len(); i++ {
+		ocmd := internaldata.ResourceMetricsToOC(rms.Get(i))
+		if len(ocmd.Metrics) == 0 {
+			continue
```

Comment:
@pjanotti do we need to count this somehow in internal metrics? This is the case when we receive a request but it contains no metrics.

Reply:
@tigrannajaryan this PR did not change this, so we probably need to file an issue to fix this.